assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -54,4 +58,15 @@ if (getQueryParameter('newDesigns', 'false') === 'true') {
   );
 }
 
+(function initialiseAmountsTest() {
+  try {
+    const defaultSelectedAmount =
+      defaultAmounts[store.getState().common.abParticipations.ukRecurringAmountsTest || 'control'];
```

Comment:
Which case is cover by `|| 'control'`? If the user is not in a variant `ukRecurringAmountsTest` will have value `notInTest` and `|| control` will be not execute.